src/99-appendices/01-contributors.md

```diff
@@ -46,7 +46,6 @@ your name is not listed, please add it.
 -   Elizabeth DuPre 📖💡🔍🤔
 -   Eric A. Earl 🤔
 -   Anders Eklund 📖📢💻
--   Oscar Esteban 📖🔧🤔💬💻
```

Comment:
Why is this line deleted?

Reply:
No idea why this line went missing, I have added Oscar back